<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>withdrawalSuccessful</class>
 <widget class="QDialog" name="withdrawalSuccessful">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(110,110,110);
text-transform: uppercase;
background-color: rgb(135, 203, 185);</string>
  </property>
  <widget class="QPushButton" name="nappi_sulje">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242,242,242);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Sulje ikkuna</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-12</x>
     <y>100</y>
     <width>711</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135, 203, 185);
color: rgb(248,248,248);
letter-spacing: 2px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Nosto suoritettu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>541</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 234, 227);
border: 5px;
border-color: rgb(135, 203, 185);
border-radius: 15px;</string>
   </property>
   <widget class="QLabel" name="label_maara">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>419</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;xxxx euroa nostettu.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_saldo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>419</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tilillä jäljellä xxxx euroa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>nappi_sulje</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
